saskatoon/harvest/urls.py

```diff
@@ -84,9 +84,9 @@
          views.TemplateView.as_view(template_name='app/property_thanks.html'),
          name='property-thanks'),
 
-    path(r'stats/',
-         views.TemplateView.as_view(template_name='app/stats.html'),
-         name='statistics'),
+    path('stats/', api.StatsView.as_view(), name='statistics'),
+
+    path('stats/<int:season>/', api.StatsView.as_view(), name='statistics'),
```

Comment:
I believe `path('stats/', api.StatsView.as_view(), name='statistics')` would be enough and the season year could be passed using the `season` field from HarvestFilter with `stats/?season=2022`

Reply:
@2ynn hi, I'm now using url parameters as you advised. 

![image](https://user-images.githubusercontent.com/52796958/166816721-b074319c-c245-4eca-9fac-b29fc18676f3.png)